Exerscise_3/Show.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "ICommand.h"
+#include "IRecord.h"
+#include <string>
+#include <vector>
+#include <memory>
+
+class Show: public ICommand {
+public:
+    std::vector<std::pair<std::string, std::vector<std::pair<int, std::unique_ptr<IRecord>>>>>  &show_container;
```

Comment:
Two comments here.
1. For such a complicated types, it's good to use aliases, e.g.: `using TablesContainer = std::vector<...>`
2. I think it'd be better to use `std::map`, not `vector`. You want to map a table name to a map of id to record:
`std::map<std::string, std::map<int, std::unique_ptr<IRecord>>>`.
Additionally, map automatically sorts entries by key, and that's exactly what you want.